src/chef-mover/src/mover_server_admins_global_group_callback.erl

```diff
@@ -47,20 +48,33 @@ migration_init() ->
 migration_action(GlobalOrgId, _AcctInfo) ->
     BifrostSuperuserAuthzId = mv_oc_chef_authz:superuser_id(),
     ErchefSuperuserAuthzId = get_erchef_superuser_authz_id(),
-    ServerAdminsAuthzId = case create_server_admins_authz_group(BifrostSuperuserAuthzId) of
-        AuthzId when is_binary(AuthzId) ->
-            AuthzId;
-        AuthzError ->
-            lager:error("Could not create new authz group for server-admins."),
-            throw(AuthzError)
-    end,
-    
-    case create_server_admins_global_group(ServerAdminsAuthzId, GlobalOrgId) of
-        {chef_sql, {GroupError, _}} ->
-	    lager:error("Could not create new erchef group for server-admins."),
-            throw(GroupError);
-	_ -> true
-    end,
+
+    %% if server-admins is already created, skip this step
+    ServerAdminsAuthzId =
+        case get_server_admins_authz_group_if_exists() of
+            ExistingServerAdminsAuthzId ->
```

Comment:
Er...does this need to go below the `needs_created` atom to match? Seems like it's matching `needs_created` based on my testing.